Document `--exclude_rule` (#393)
Certora/Documentation: docs/prover/cli/options.md

In the last example the `transfer` method of all contracts in the
scene will be used, but only the `deposit` method of the primary contract.

(--exclude_method)=
## `exclude_method`

**What does it do?**
This option is the opposite of {ref}`--method` and will exclude from parametric
rules and invariants any method mentioned in this list. The methods and their
contracts are interpreted in the same way as they are in the {ref}`--method`
option.

**When to use it?**

> If there are specific methods for which a rule/invariant takes too long to run
> or require a different configuration.

Reviewer: ```suggestion
If there are specific methods for which a rule/invariant takes too long to run
or require a different configuration, this config option allows to exclude it from verification.
```

```{note}
If a method is listed by both {ref}`--method` and {ref}`--exclude_method`,

> exclusion takes precedence, this option allows to exclude it from verification.

Reviewer 2: ```suggestion
exclusion takes precedence.
```

Reviewer: I reverted it for sure.

```

**Example**
```sh
certoraRun --method '_.deposit(uint)' --exclude_method 'C.deposit(uint)'
```

> This will include all `deposit(uint)` methods in the scene, with the exception of the

Reviewer 2: ```suggestion
This will include all `deposit(uint)` methods in the scene except the
```
